packages/yourloops/test/integration/pages/home-page/caregiver-home-page.spec.tsx

```diff
@@ -140,8 +145,9 @@ describe('Caregiver home page', () => {
     const patientData = patientTableBody.getByText(patientFullName)
     expect(patientData).toBeVisible()
 
-    const patientRow = screen.queryByTestId(`patient-row-${unmonitoredPatient.userId}`)
-    const removePatientButton = within(patientRow).getByRole('button', { name: 'Remove patient ylp.ui.test.patient28@diabeloop.fr' })
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
It feels weird having to disable eslint in some code that should be relatively clean, if this rule is so painful maybe it could be deactivated for the whole project? (I know I've struggled with this one too not long ago)

Reply:
Removing this line seems fine actually. I don't know why my IDE was complaining before...